app/src/routes/event/unregister/[token]/+page.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  import { Alert } from "flowbite-svelte";
+
+  export let data;
+</script>
+
+<section class="w-full mt-2 mb-80 mx-0">
+  {#if data?.success}
+    <Alert color="green" class="mb-6">Du har avregistrert deg fra arrangementet!</Alert>
+  {:else}
+    <Alert color="red" class="mb-6">Vi kunne ikke avregistrere deg, prøv på nytt senere!</Alert>
+  {/if}
+</section>
```

Comment:
Får ikke opp dette når jeg prøver å avmelde meg 🤔 Det var vel meningen?

Reply:
Jo det skal være mulig! Når du er meldt på et arrangement og trykker avmelding så skal du få en URL med en JWT token som du kan bruke for å teste!

Reply:
Endrer du eller skriver du en feil JWT token så skal du få en feilmelding

Reply:
Skjønner, det funket ja!